src/main/java/core/basesyntax/figures/FigureData.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.figures;
+
+public interface FigureData {
+    void area();
```

Comment:
```suggestion
    void getArea();
```

methods names should starr from verbs https://mate-academy.github.io/style-guides/java/java.html#s5.2.3-method-names